### DISCUSSION TITLE
encrypting answers in application form

### DISCUSSION THREAD

**Discussion by thelostone-mc** [gitcoinco/grants-stack#140] [Q&A] [answered]
### Description

How does grants-hub encrypt PII info and send it to round manager ? 

> In general for us, if the question has an encrypted: true field  and a publicKey field, we can encrypt with that pk or a derived one and save the encrypted answer (we can think about the algorithm later). but I'm not sure what the operator can do to decrypt it. metamask has some deprecated methods to do that, but they are actually not really supported by other wallets.

**Comment by kevinrolsen** [comment, marked as answer]
I would highly recommend we look into LIT protocol. This is a perfect use case for their tech. https://developer.litprotocol.com/docs/introduction/whatislitprotocol/

**Reply by thelostone-mc** [reply]
@nfrgosselin Once we finalize if email is deemed PII , this is something we should look into and discuss with the grant-hub team if this would work for us. Would require a quick spike in that case for the team to look into LIT

**Reply by codenamejason** [reply]
Currently doing a write-up on Lit Protocol and use cases as well as investigating how Passport is storing PII.
